postgresql/gen-yaml/gen.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o nounset
+
+mkdir -p manifests/upstream
+rm -f manifests/upstream/*
+
+# Fetch from helm repo: https://github.com/bitnami/charts/tree/main/bitnami/postgresql
+helm repo add bitnami https://charts.bitnami.com/bitnami
+
+helm template "postgresql" bitnami/postgresql --version "${BITNAMI_POSTGRES_RELEASE}" \
+  --set fullnameOverride="postgresql" \
+  --set nameOverride="postgresql" \
+  --set commonAnnotations."app\.uw\.systems\/description"="" \
+  --set commonAnnotations."app\.uw\.systems\/tier"="" \
+  --set commonAnnotations."app\.uw\.systems\/repos"="https://github.com/utilitywarehouse/shared-kustomize-bases/tree/main/postgresql" \
+  --set auth.existingSecret="postgresql" \
+  --set auth.secretKeys.adminPasswordKey="password" \
+  --set auth.secretKeys.userPasswordKey="user-password" \
+  --set global.postgresql.auth.database="database" \
+  --set global.postgresql.auth.username="testuser" \
+  --set primary.resources.requests.cpu="0" \
```

Comment:
I would propose to set the resources in the kustomize override- there is no point to regenerate manifest on tweaking memory limit